src/Raven.Client/Documents/Changes/DatabaseChanges.cs

```diff
@@ -450,6 +450,7 @@ private DatabaseConnectionState GetOrAddConnectionState(string name, string watc
                             await UnmanagedJsonParserHelper.ReadObjectAsync(builder, stream, parser, buffer).ConfigureAwait(false);
 
                             var json = builder.CreateReader();
+                            json.BlittableValidation();
```

Comment:
There is no need to do this here, we just parse this as a _string_, so we know that this is valid.